src/mds/QuiesceDbManager.h

```diff
@@ -250,7 +250,7 @@ class QuiesceDbManager {
     struct PeerInfo {
         QuiesceMap diff_map;
         QuiesceTimePoint last_activity;
-        QuiesceDbVersion last_sent_version;
+        QuiesceDbVersion last_sent_version{};
```

Comment:
@leonid-s-usov this was missed too. Please re-approve if you think this is good.

Reply:
I would prefer to have the change at the QuiesceDbVersion struct definition site. All we need to do is to provide initializers to the epoch and the set_version.

```
struct QuiesceDbVersion {
  epoch_t epoch = 0;
  QuiesceSetVersion set_version = 0;
  auto operator<=>(QuiesceDbVersion const& other) const = default;
  QuiesceDbVersion& operator+(unsigned int delta) {
    set_version += delta;
    return *this;
  }
};
```

With this approach, people using the type won't have to worry about initializing it explicitly.